recipes/unishox2-py3/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "unishox2-py3" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/unishox2-py3-{{ version }}.tar.gz
+  sha256: cd2595de196ffd8b93bea7a76f5796d8293f32d1f7c9d3660af0b764ab917963
+
+build:
```

```suggestion
build:
  ignore_run_exports_from:
    - {{ compiler('cxx') }}
```
Because [the standard library isn't used.](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=609451&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=2819)